lib/dev-server.js

```diff
@@ -94,7 +94,7 @@ async function devServer(options) {
         app.use(fallback(entryPoint, { root: options.contentBase }));
     }
 
-    server.listen(options.port);
+    server.listen(options.port, options.host);
 
     console.log(`[Nollup] Listening on ${options.https ? 'https' : 'http'}://localhost:${options.port}`);
```

Comment:
```suggestion
    console.log(`[Nollup] Listening on ${options.https ? 'https' : 'http'}://${options.host}:${options.port}`);
```

There was no report on running with a different host, need to display the host IP to the user